mercantile/__init__.py

```diff
@@ -494,18 +507,19 @@ def parent(*tile, **kwargs):
     """
     tile = _parse_tile_arg(*tile)
 
+    x, y, z = tile
+
+    if z == 0:
+        raise ParentTileError("root tile does not have a parent")
```

Comment:
Let me know if this makes sense to you folks, or if we should return `None` instead here for the root tile's parent.

`None` could be a bit more elegant in some code walking up the hierarchy as in

    while (tile := parent(tile))
        fn(tile)

:shrug:

Reply:
returning `None` for exactly that case seems pragmatic :+1: